test/updaters/module-version.ts

```diff
@@ -0,0 +1,40 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {readFileSync} from 'fs';
+import {resolve} from 'path';
+import * as snapshot from 'snap-shot-it';
+import {describe, it} from 'mocha';
+import {ModuleVersion} from '../../src/updaters/terraform/module-version';
+
+const fixturesPath = './test/updaters/fixtures';
+
+describe('versions.tf', () => {
```

Comment:
@bharathkkb Can we add a test fixture for a submodule?

Reply:
@morgante sg, currently logic for submodule is in the releaser so we may need to add a test there.
https://github.com/googleapis/release-please/blob/25a9b5296831d147e24e1239eef770188a874601/src/releasers/terraform-module.ts#L90

Reply:
@bharathkkb when adding tests for releasers, take a look at these tests:

https://github.com/googleapis/release-please/blob/master/test/releasers/simple.ts#L43

I've been switching to using sinon and mocking for tests, rather than nock (_which requires you enumerate every HTTP endpoint you hit with the GitHub API). I think you'll find sinon easier.

Reply:
@morgante @bcoe added some tests primarily testing the behavior for modules, modules+sub modules and backward compatibility if no versions.tf or module versions defined.